app/renderer/task_config.js

```diff
@@ -74,7 +74,7 @@ module.exports = {
         return data.map((suite) => {
             let result = new Suite(suite.title);
             result.tasks = suite.tasks.map((task) => {
-                return new Task(task.title, task.path, task.command);
+                return new Task(task.title, task.path, task.command, task.order);
```

Comment:
I understand that task.order no longer exists. Is this here for some reason or is a typo?